### PR TITLE
change data reviewer link to data approval link

### PR DESCRIPTION
**Pull request by deeonwuli**
### :pushpin: References

* **Issue:** Closes https://app.clickup.com/t/3dfg9gc

### :art: Screenshots
Screenshot 2022-09-07 at 15 20 50

### CODE REVIEW

**Comment by adrianq** [comment]
@deeonwuli Does the data approval URL exist? I am getting this URL:
http://localhost:8080/data-approval/HPGEAQ1dPJQ/actual/202208
while I should be getting something like:
http://localhost:8080/api/apps/Data-Management-App/index.html#/data-approval/HPGEAQ1dPJQ/actual/202208

Can you please have a look at this tomorrow morning? We are aiming to deploy this new version tomorrow

File: src/models/ProjectNotification.ts [hunk 1]
```diff
@@ -57,7 +57,18 @@ export class ProjectNotification {
...
+        const projectId = this.project.id;
+        const dataApprovalLink =
+            pageOrigin +
+            "/api/apps/Data-Management-App/index.html#/data-approval/" +
+            projectId +
+            "/" +
+            dataSetType +
+            "/" +
+            period;
+
```
Comment (tokland) [inline src/models/ProjectNotification.ts:71]: We have `getUrl` as an example of how to generate programmatically the route and make it work for DEV and APP (with features involving links, check always that the app works). I've sent you a proposal, check it and upload with the changes you see fit.

**Review by tokland — CHANGES_REQUESTED** [review]

**Review by tokland — APPROVED** [review]
All good!

**Comment by adrianq** [comment]
Thanks!

**Pull request merged**